### PR TITLE
update resource browser docs

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: content/en/user-guide/web-application/resource-browser/index.md [hunk 1]
```diff
@@ -5,60 +5,70 @@ description: >
...
 
-- To view the running services in your LocalStack container, you can use the [System Status](https://app.localstack.cloud/status). This view provides a quick overview of the running services in your LocalStack container. To get a detailed telemetry of your API calls to your LocalStack containers, check out [Stack Insights](https://docs.localstack.cloud/user-guide/web-application/stack-insights/).
+{{<alert title="Switching AWS regions">}}
+An AWS region dropdown menu in the dashboard is located on the top right of the page. You can select your desired region to ensure that you can view your resources. If you cannot view resources that you have recently created, you should verify that you are checking the resources in the correct region.
 {{< /alert >}}
 
 ## Supported services
 
-We currently support the following local AWS services in our Resource Browser:
+The Resource Browser supports the following AWS services:
```
Comment (Pive01) [inline content/en/user-guide/web-application/resource-browser/index.md:24]: Should we make some sort of auto-script that mutates this table automatically when a new RB is added?

File: content/en/user-guide/web-application/resource-browser/index.md [hunk 2]
```diff
@@ -5,60 +5,70 @@ description: >
...
+The Resource Browser provide an experience similar to the AWS Management Console. However, the Resource Browser is not a replacement for the AWS Management Console and only replicate some of the features of the AWS Management Console. We recommend using our [integrations](https://docs.localstack.cloud/user-guide/integrations/) to create your resources, with the Resource Browser being used for quick viewing and management of your resources.
 
-- The Resource Browser provide an experience similar to the AWS Management Console. However, the Resource Browser is not a replacement for the AWS Management Console and only replicate some of the features of the AWS Management Console. We recommend using our [integrations](https://docs.localstack.cloud/user-guide/integrations/) to create your resources, with the Resource Browser being used for quick viewing and management of your resources.
+The LocalStack Web Application connects to your LocalStack container and retrieves the information about your local resources directly via the `localhost` without using the internet. None of the information is sent to the internet, or stored on any external servers maintained by LocalStack.
```
Comment (Pive01) [inline content/en/user-guide/web-application/resource-browser/index.md:16]: I would remove `the "localhost"` in favour of just `localhost`

File: content/en/user-guide/web-application/resource-browser/index.md [hunk 3]
```diff
@@ -5,60 +5,70 @@ description: >
...
 
+## Introduction
+
 The LocalStack Resource Browser allow you to view, manage, and deploy AWS resources locally while building & testing their cloud applications locally. It provides an internal, integrated experience, similar to the AWS Management Console, to manage the ephemeral resources locally in a LocalStack container on your local machine.
 
 <img src="resource-browser.png" alt="LocalStack Web Application's Resource Browsers outlining various local AWS services" title="Resource Browser" width="900" />
 
-{{< alert title="Warning" color="warning">}}
-- If you encounter a `Network Failure` error message while accessing the Resource Browser, it is likely that the LocalStack container is not running or the instance is not reachable at the endpoint specified in the instance bookmark.
-{{< /alert >}}
```
Comment (Pive01) [inline content/en/user-guide/web-application/resource-browser/index.md:14]: Have we removed this? Feel like is helpful

**Review by Pive01 — APPROVED** [review]
LGTM! just a nit

File: content/en/user-guide/web-application/resource-browser/index.md [hunk 4]
```diff
@@ -5,60 +5,70 @@ description: >
     The Resource Browser allows you to view and manage your local AWS resources through the LocalStack Web Application.
 ---
 
+## Introduction
+
 The LocalStack Resource Browser allow you to view, manage, and deploy AWS resources locally while building & testing their cloud applications locally. It provides an internal, integrated experience, similar to the AWS Management Console, to manage the ephemeral resources locally in a LocalStack container on your local machine.
```
Comment (lukqw) [inline content/en/user-guide/web-application/resource-browser/index.md:10]:
```suggestion
The LocalStack Resource Browser allow you to view, manage, and deploy AWS resources locally while building & testing their cloud applications locally. It provides an internal, integrated experience, similar to the AWS Management Console, to manage the ephemeral resources in a LocalStack container on your local machine.
```

File: content/en/user-guide/web-application/resource-browser/index.md (hunk 3 above)
Comment (lukqw) [reply content/en/user-guide/web-application/resource-browser/index.md:14]: It's at the very bottom now

**Review by lukqw — APPROVED** [review]
Thanks for the strong revision on this part of the docs @HarshCasper! 💪

File: content/en/user-guide/web-application/resource-browser/index.md (hunk 1 above)
Comment (HarshCasper) [reply content/en/user-guide/web-application/resource-browser/index.md:24]: Yes, but let's keep it as a future item 👍

**Pull request merged**